### ISSUE TITLE
Parameter 'right' is not used inside function body. hb-ot-kern-table.hh 115

### ISSUE THREAD

**Issue by kojiishi** [harfbuzz/harfbuzz#728]
From [crbug.com/805881](http://crbug.com/805881):

V751 Parameter 'right' is not used inside function body. hb-ot-kern-table.hh 115

```c++
inline int get_kerning (hb_codepoint_t left,
                        hb_codepoint_t right,
                        const char *end) const
{
  unsigned int l = (this+leftClassTable).get_class (left);
  unsigned int r = (this+leftClassTable).get_class (left);  // <=
  unsigned int offset = l * rowWidth + r * sizeof (FWORD);
  ....
}
```

**Comment by bungeman** [comment]
Opened merge request #729 with a proposed fix.

**Comment by ebraminio** [comment]
Closed as #729
